pom.xml

```diff
@@ -125,7 +125,7 @@
         <!-- Please don't upgrade the version to 4.10+, it depends on JDK 11 -->
         <antlr4.version>4.9.3</antlr4.version>
         <antlr.st4.version>4.3.4</antlr.st4.version>
-        <apache.archive.dist>https://archive.apache.org/dist</apache.archive.dist>
+        <apache.archive.dist>https://dlcdn.apache.org</apache.archive.dist>
```

Comment:
does this work for archived versions?

Reply:
<img width="906" alt="image" src="https://github.com/user-attachments/assets/b2f35882-0ced-4c4e-9e30-59d7d39e408a">

Not sure whether the network is reachable, need to test.

Reply:
ok, there is no `archived` version ...

Reply:
BTW, the tgz should be loaded from the cache directly, unless the recent plugin bumping https://github.com/apache/kyuubi/pull/6807 causes some issues

Reply:
Thanks for the info.